sys/include/net/nanocoap.h

```diff
@@ -498,6 +520,20 @@ size_t coap_put_option_block1(uint8_t *buf, uint16_t lastonum, unsigned blknum,
  */
 size_t coap_put_block1_ok(uint8_t *pkt_pos, coap_block1_t *block1, uint16_t lastonum);
 
+/**
```

Comment:
Please change to coap_opt_put_block2() to be consistent with #9085. We'll need to change the equivalent block1 function name in a follow-on PR.

Reply:
I'm also changing the `coap_get_block2` to `coap_opt_get_block2`

Reply:
coap_opt_put block2() looks good. See my comment on coap_opt_get_block2() though.